packages/theme/css/characteristics/target-next.css

```diff
@@ -0,0 +1,4 @@
+.salt-theme.salt-theme-next {
+  --salt-target-background-hover: var(--salt-palette-accent-default);
```

Comment:
```suggestion
  --salt-target-background-hover: var(--salt-palette-accent);
```

Reply:
picked this up running validation on https://github.com/jpmorganchase/salt-ds/pull/3143